manifests/init.pp

```diff
@@ -160,6 +160,8 @@
 #
 # $cors_domains::                 List of domains that show be allowed for Cross-Origin Resource Sharing
 #
+# $trusted_proxies::              List of trusted proxies (at least every smart-proxy)
```

Comment:
Could this be named to include that they are http(s) proxies? I worry about confusion with the ole smart-proxy which is what I thought this was aimed at at first.

Reply:
It fixes: https://community.theforeman.org/t/using-cloud-init-templates-with-foreman-proxy-foreman-proxy-content/20269/20

Should I change the description or the key?

Reply:
I would say the description because `$trusted_proxies` matches the setting in the settings file.

Reply:
I still consider this unresolved. This is ambiguous because there's multiple proxies. This should explicitly say that these are reverse HTTP proxies.

The Smart Proxy *can* act as a reverse HTTP proxy, but only if you use some modules. Templates is one example here (and probably the most important).

In the Katello case, there is also a reverse HTTP proxy for RHSM and there it may also be important for correctly logging the remote IP. In that case Apache acts as a reverse proxy.

Reply:
Ok. If you have something in mind which I should write just let me know. Any suggestions is welcome 😁

Reply:
Changed the description again.